webviz_subsurface/plugins/_pvt_plot.py

```diff
@@ -0,0 +1,640 @@
+                                dcc.Dropdown(
+                                    id=self.uuid("ensemble"),
+                                    clearable=False,
+                                    multi=True,
+                                    options=[
+                                        {"label": i, "value": i} for i in self.ensembles
+                                    ],
+                                    value=self.ensembles[0],
+                                ),
+                            ],
+                        ),
+                        html.Label(
+                            id=self.uuid("phase_selector"),
+                            children=[
+                                html.Span("Phase:", style={"font-weight": "bold"},),
+                                dcc.Dropdown(
+                                    id=self.uuid("phase"),
+                                    clearable=False,
+                                    options=[
+                                        {"label": i.lower().capitalize(), "value": i,}
+                                        for i in self.phases
+                                    ],
+                                    value=list(self.phases)[0],
+                                ),
+                            ],
+                        ),
+                        dcc.Store(id=self.uuid("stored_pvtnum"), data={}),
+                        html.Label(
+                            id=self.uuid("pvtnum_selector"),
+                            children=[
+                                html.Span("Pvtnum:", style={"font-weight": "bold"}),
+                                dcc.Dropdown(
+                                    id=self.uuid("pvtnum"),
+                                    clearable=False,
+                                    multi=True,
+                                    options=[
+                                        {"label": i, "value": i} for i in self.pvtnums
+                                    ],
+                                    value=self.pvtnums[0],
+                                ),
+                            ],
+                        ),
+                    ],
+                ),
+                html.Div(
+                    id=self.uuid("graphs"),
+                    style={"flex": "4"},
+                    children=wcc.Graph(id=self.uuid("graph")),
+                ),
+            ],
+        )
+
+    def set_callbacks(self, app):
+        @app.callback(
+            Output(self.uuid("graph"), "figure"),
+            [
+                Input(self.uuid("color_by"), "value"),
+                Input(self.uuid("phase"), "value"),
+                Input(self.uuid("ensemble"), "value"),
+                Input(self.uuid("pvtnum"), "value"),
+            ],
+        )
+        def _update_graph(
+            color_by: str, phase: str, ensembles: List[str], pvtnums: List[str]
```

Comment:
As you support conversion from a str to a list further down
```suggestion
            color_by: str, phase: str, ensembles: Union[List[str], str], pvtnums: Union[List[str], str]
```